src/enzo/Grid_TestRadiatingStarParticleInitializeGrid.C

```diff
@@ -0,0 +1,393 @@
+/***********************************************************************
+/
+/  GRID CLASS (INITIALIZE THE GRID FOR A RADIATING STAR PARTICLE TEST)
+/
+/  written by: Greg Bryan
+/  date:       June, 2012
+/  modified1:  Danielle Skinner
+/  date:       July, 2022
+/
+/  PURPOSE:
+/
+/  RETURNS: FAIL or SUCCESS
+/
+************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+#include "ErrorExceptions.h"
+#include "macros_and_parameters.h"
+#include "typedefs.h"
+#include "global_data.h"
+#include "Fluxes.h"
+#include "GridList.h"
+#include "ExternalBoundary.h"
+#include "Grid.h"
+#include "phys_constants.h"
+
+int GetUnits(float *DensityUnits, float *LengthUnits,
+	     float *TemperatureUnits, float *TimeUnits,
+	     float *VelocityUnits, double *MassUnits, FLOAT Time);
+
+// Used to compute Bonner-Ebert density profile
+double ph_BE(double r);
+double ph_q(double r);
+double ph_Ang(double a1, double a2, double R, double r);
+
+// Returns random velocity from Maxwellian distribution
+double ph_Maxwellian(double c_tilda, double vel_unit, double mu, double gamma);
+
+static int PhotonTestParticleCount = 0;
+
+int grid::TestRadiatingStarParticleInitializeGrid(float TestStarParticleStarMass, 
+					 float *Initialdt,
+					 FLOAT TestStarParticleStarVelocity[],
+					 FLOAT TestStarParticleStarPosition[],
+           float TestStarParticleDensity,
+           float TestStarParticleEnergy,
+           float TestStarParticleVelocity[],
+           float TestStarParticleBField[],
+           float InitialTemperature,
+           float PhotonTestInitialFractionHII, 
```

Comment:
Change all of the `PhotonTest` variables to start with `TestStarParticle` to be consistent with other parameters.